diktat-rules/src/test/resources/test/paragraph3/indentation/IndentationFull1Test.kt

```diff
@@ -46,4 +46,17 @@ fun foo(
             .bar()
         }"
     }
+
+    val dockerFileAsText =
+                """
+                        FROM $baseImage
```

Comment:
@aktsay6 the issue was about a bug with disappearing spaces. We shouldn't change indentation of different lines of a string, because this way we obviously change the string itself. We could only move all strings at once, if there is `trimIdent` or `trimMargin`.